spotbugs/etc/messages.xml

```diff
@@ -8785,11 +8785,29 @@ Using floating-point variables should not be used as loop counters, as they are
       </p>]]>
     </Details>
   </BugPattern>
+
+  <BugPattern type="IAOM_DO_NOT_INCREASE_METHOD_ACCESSIBILITY">
+    <ShortDescription>Accessibility of an overridden or hidden method was increased.</ShortDescription>
+    <LongDescription>Accessibility of an overridden or hidden method {1} was increased from {2} to {3}.</LongDescription>
```

Comment:
I think the `an` is unnecessary here, since the method is explicitly named.

Reply:
Nice catch! I appreciate precision like that!